### DISCUSSION TITLE
Hori GC Controller recognized as xbox360

### DISCUSSION THREAD

**Discussion by Ataraxia13** [cathery/sys-con#133] [Troubleshooting]
**Description**
While connecting the HORI gamecube-style pro controller, the switch recognize him as an Xbox360 controller, making me crash while playing Xenoblade Chronicles 2 like when I use my DS3, I even had to remap it in the config and I tried removing the xbox360.ini to see if it would have been recognized normally but nope, I had to remove sys-con from the sd, is there a way to make it work properly as a normal pro controller connected via usb?

**Environment**
- 9.1.0 - OFW
- 0.10.1 - CFW Atmosphere
- 0.5.2 - sys-con

I'll leave an image of the controller I'm talking about
![81w32FHyNML _SL1500_](https://user-images.githubusercontent.com/60212080/72964102-35424100-3db9-11ea-812d-f591316e1dee.jpg)

**Comment by cathery** [comment]
I'm sorry you are experiencing this. For now when using Hori GC it's best you disable sys-con altogether. Can you please scan the controller using this app https://files.catbox.moe/re60qo.rar and send me a screenshot? I could make the sysmodule avoid Hori GC controllers with that info.

**Comment by Ataraxia13** [comment]
> 
> 
> I'm sorry you are experiencing this. For now when using Hori GC it's best you disable sys-con altogether. Can you please scan the controller using this app https://files.catbox.moe/re60qo.rar and send me a screenshot? I could make the sysmodule avoid Hori GC controllers with that info.

App is not working at all, atleast if I have to press Y from the Hori GC. I can't try by pressing via joycon because my right joycon doesn't work anymore

**Comment by cathery** [comment]
1. Disconnect all your controllers
2. Connect your joycons
3. Connect the Hori GC
4. Open the app and press Y using joycons
5. This should print out the info of all USB connected devices, screenshot that and send it here

**Comment by Ataraxia13** [comment]
I'll have to try this whenever I buy another right joycon, hope to get it for Valentine's Day from gf, ty for the tutorial, I'll reply if I get the possibility to try this before Valentine
Thank you

**Comment by cathery** [comment]
If you can navigate using the Hori GC, disconnect all controllers and connect only the Hori GC, then press Y using Hori GC.

**Comment by Ataraxia13** [comment]
Hori GC doesn't work while playing the app

**Comment by cathery** [comment]
Give me a moment, I'll make a change to the app

**Comment by cathery** [comment]
I've updated it to accept most buttons from any controller, give it a try https://files.catbox.moe/kym7hc.zip

**Comment by Ataraxia13** [comment]
Had some problem finding the Album folder of the EmuMMC, but here is the screenshot
![2020012311532100-DB1426D1DFD034027CECDE9C2DD914B8](https://user-images.githubusercontent.com/60212080/72979353-6da54780-3dd8-11ea-9de9-9da5130c7e2f.jpg)

**Comment by cathery** [comment]
The device in the screenshot should not be recognized as an xbox 360 controller by any means so that's very weird. But thank you for the info.

**Comment by Ataraxia13** [comment]
Idk how this works, but when connected with sys-con, it is recognized as the Xbox360 controller, I was even able to remap the buttons, because A-B,X-Y,L-ZL and R-ZR where swapped.
Anyway thank you for the consideration and keep going, have a nice day bud

**Comment by cathery** [comment]
For the xenoblade 2 crash please see this: https://github.com/cathery/sys-con/issues/91#issuecomment-577752073

**Comment by samuraivolon** [comment]
> Idk how this works, but when connected with sys-con, it is recognized as the Xbox360 controller, I was even able to remap the buttons, because A-B,X-Y,L-ZL and R-ZR where swapped.
> Anyway thank you for the consideration and keep going, have a nice day bud

I have the same controller (Hori BattlePad for Switch), i tryed remapping the buttons but it's not working for me.. can you send me a screen with your config_xbox360.ini configuration?? 
Here's mine
![image](https://user-images.githubusercontent.com/52016532/74027744-893b5100-4987-11ea-8c28-4df0e26ae0c4.png)
